### ISSUE TITLE
export images with time and date captured

### ISSUE THREAD

**Issue by xvids121** [sabrogden/Ditto#10]
Unable to find a way to export images with the timestamp of the date and time that each image was actually captured. 

There does not appear to be an option to include a timestamp.  It appears that the metadata of each exported image is the same - the date and time at export.

I take a lot of screenshots and having the date and time of the image taken is critical if I need to be able to go back and perform research.

**Comment by sabrogden** [comment]
Are you asking for the created date time on the exported file to be set to when the clip was originally copied?

**Comment by xvids121** [comment]
> 
> 
> Are you asking for the created date time on the exported file to be set to when the clip was originally copied?

Yes the date and time the clipboard captured the image.  The "Date Added" attribute when listing the "properties" of a clip (Alt+Enter) appears to be the appropriate value.

**Comment by xvids121** [comment]
Hello Sabrogden,

How do I work out when or if this feature could be added?

**Comment by xvids121** [comment]
Hi Sabrogden,

Are there any plans or manual techniques / methods to be able to export images with an appropriate timestamp?   Perhaps this could eventually expand to include a user-definable out file naming scheme where the user can combine what they want, including date captured and other metabata?

**Comment by szegi** [comment]
user-definable out file naming scheme would be helpful:)

**Comment by xvids121** [comment]
Hi Anyone,

Perhaps the functionality is already available somehow?  Is there a process for exporting images from database?   I have a lot of stuff in a 50gb database and need to find a way to export it.  Can someone perhaps suggest a process they have tested themselves or believe should work okay?
